main.py

```diff
@@ -1,12 +1,199 @@
+            response = retro_video_store.create_video(title=title, release_date=release_date, total_inventory=total_inventory)
+            print_hashtags()
+            print("New Video information:", response["id"])
+
+        elif choice=='9':
+            print(f"Time to update a video's information!: {retro_video_store.selected_video}")
+            title=input("What is it's new name? ")
+            release_date=input("What is their new release date? ")
+            total_inventory=input("How many copies of this do we have? ")
+            response = retro_video_store.update_video(title=title, release_date=release_date, total_inventory=total_inventory)
+
+            print_hashtags()
+            print("Updated video info:", response["title"])
+
+        elif choice=='10':
+            video_to_delete = input("What is the id of the video you wish to delete? ")
+            if video_to_delete.isnumeric():
+                video_to_delete = int(video_to_delete)
+                retro_video_store.delete_video(video_to_delete)
+
+            print_hashtags()
+            print("That video has been deleted.")
+
+            print_hashtags()
+            print("Here are the remaining videos in store......")
+            print(retro_video_store.list_of_videos())
+
+        elif choice=='11':
+            print("Be prepared to enter the id of the video and customer for check-out.")
+            video_id = input("What is the video id? ")
+            if video_id.isnumeric():
+                video_id = int(video_id)
+                retro_video_store.get_video(id=video_id)
+            
+            customer_id = input("What is the customer is? ")
+            if customer_id.isnumeric():
+                customer_id = int(customer_id)
+                retro_video_store.get_customer(id=customer_id)
+
+            response = retro_video_store.check_out_video(customer_id, video_id)
+            print_hashtags()
+            print("Video Check-out information: ", response)
+
+        elif choice=='12':
+            print("Be prepared to enter the id of the video and customer for check-in.")
+            video_id = input("What is the video id? ")
+            if video_id.isnumeric():
+                video_id = int(video_id)
+                retro_video_store.get_video(id=video_id)
+            
+            customer_id = input("What is the customer is? ")
+            if customer_id.isnumeric():
+                customer_id = int(customer_id)
+                retro_video_store.get_customer(id=customer_id)
+
+            response = retro_video_store.check_in_video(customer_id, video_id)
+            print_hashtags()
+            print("Video Check-in information: ", response)
+
+        elif choice=='13':
+            list_options()
+
+        elif choice=='14':
+            play=False
+            print("\nThanks for visiting the Retro Video Store! Come again :)")
+
+        print_hashtags()
+
+run_cli()
```

Comment:
Nice work displaying all the options here and using the example Task List CLI as a template for this project.

A future refactor would include separating out the logic for processing the user input (what's inside your conditionals) from the logic to display menu options. The logic inside each conditional can be turned into helper methods created in a separate file so that you call. For example:


In a new`customer_routes.py` file:

```
from video_store import Video_Store

class Customer:
    def __init__(self):
        self.retro_video_store = Video_Store(url="http://127.0.0.1:5000")
        
    def get_all_customers():
           for customer in retro_video_store.list_of_customers():
                    print(customer)
                    
    def get_one_customer():
         
            id=input("What is the id of the customer you want information about? ")
            if id.isnumeric():
                id=int(id)
                print("Here is this customers information:", retro_video_store.get_customer(id=id))
            else:
                print("Please enter the number id.")
```

Your main.py file would refer to this new function inside of `customer_routes.py` like so:

```
from customer_routes import Customer

def main:
     while play == True:
             if choice == '1':
                print_hashtags()
                print("Here are all of the customers we have in our database.")
                Customer.get_all_customers()
            elif choice == '2':
                Customer.get_one_customer()
```